front/src/common/OSRDMenu.tsx

```diff
@@ -9,10 +9,11 @@ export type OSRDMenuItem = {
 type OSRDMenuProps = {
   menuRef: React.RefObject<HTMLDivElement>;
   items: OSRDMenuItem[];
+  style?: React.CSSProperties;
```

Comment:
If this is exposed to allow callers to position the menu, should we instead add a `position` property? That would deter callers from (ab)using `style` for things which should stay in CSS files.

Reply:
We also need to set the width, it's not always the same depending of who's calling the menu. I made the call to pass this props instead of pass `width` and `position`. You would prefer that ?

Reply:
I think it would be cleaner to specify width and position, this way callers know what they can and should provide

Reply:
Forgot to update here but on the second fixup, I put the width in the css directly.

For the position part, I'll probably have to put the menu inside osrd-ui directly so we might not need this props anymore. I'll try that tomorrow.